### DISCUSSION TITLE
Is CefSharp good for extracting information or should we use HtmlAgilityPack?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [cefsharp/CefSharp#4334] [Q&A] [answered]
Hello. I am first time using CefSharp .
Lets say I have loaded the document and I want to iterate inside all certain class divs and extract certain parts of information (like visible text) with their selector id class etc

How can I do that?

I know how to do with HtmlAgilityPack but it is possible and easy with CefSharp  as well? Any examples tutorials etc?

.Net Core 7, CefSharp.OffScreen

**Comment by amaitland** [comment, marked as answer]
Already asked at https://stackoverflow.com/questions/74715905/is-cefsharp-good-for-extracting-information-or-should-we-use-htmlagilitypack

Please continue the conversation there.

**Reply by amaitland** [reply]
See also https://github.com/cefsharp/CefSharp/discussions/4293
